<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductUI</class>
 <widget class="QWidget" name="ProductUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="QTableView" name="MainTableView">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolBox" name="PToolBox">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="ProductAdd">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>387</width>
         <height>397</height>
        </rect>
       </property>
       <attribute name="label">
        <string>product add/delete</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="PIDlabel">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>PID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="PIDLineEdit"/>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="NameLabel">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="NameLineEdit"/>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="PriceLabel">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>cost</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="PriceLineEdit"/>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="CntLabel">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>count</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="CntLineEdit"/>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="ImageLabel">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>150</height>
           </size>
          </property>
          <property name="text">
           <string>preview</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="ImageButton">
          <property name="text">
           <string>add image</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="AddButton">
          <property name="text">
           <string>add / change product</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QPushButton" name="DeleteButton">
          <property name="text">
           <string>delete product</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ProductSearch">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>387</width>
         <height>397</height>
        </rect>
       </property>
       <attribute name="label">
        <string>product search</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableView" name="SearchTableView">
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <widget class="QComboBox" name="SearchComboBox">
           <item>
            <property name="text">
             <string>id</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>name</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>price</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>cnt</string>
            </property>
           </item>
          </widget>
          <widget class="QLineEdit" name="SearchLineEdit"/>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="SearchButton">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
